components/disk/kmsg_matcher.go

```diff
@@ -10,10 +10,31 @@ const (
 	eventNoSpaceLeft   = "no_space_left"
 	regexNoSpaceLeft   = `No space left`
 	messageNoSpaceLeft = `no disk space left`
+
+	// e.g.,
+	// md/raid0md0: Disk failure on nvme0n1p1 detected, failing array.
+	eventRAIDArrayFailure   = "raid_array_failure"
+	regexRAIDArrayFailure   = `md/raid.*: Disk failure on .* detected, failing array`
+	messageRAIDArrayFailure = `RAID array has failed due to disk failure`
+
+	// e.g.,
+	// EXT4-fs (md0): Remounting filesystem read-only
+	eventFilesystemReadOnly   = "filesystem_read_only"
+	regexFilesystemReadOnly   = `EXT4-fs.*Remounting filesystem read-only`
```

Comment:
if the fs is not EXT4, the regex match will fail. IMO, we can just match `.*Remounting filesystem read-only`

Reply:
@eahydra Fixed